styleguide/source/_patterns/02-organisms/03-sections/02-card-row-ec/01-card-row-ec-three-up.twig

```diff
@@ -1,13 +1,26 @@
 <div class="layout layout_three_up">
-  <div class="layout_three_up-row">
-    <div class="layout_item layout_three_up-primary">
-      {% include 'molecules-card-image-landscape' %}
+    <div class="layout_three_up-row">
+        {% if cardType == 'landscape' %}
```

Comment:
Addressed in [this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/42856000e883b34e26e6d70712e55897e751ef76).